src/VisualStudio/IntegrationTest/IntegrationTests/Workspace/WorkspacesNetCore.cs

```diff
@@ -49,6 +49,9 @@ public override void MetadataReference()
             // https://github.com/dotnet/roslyn/issues/34264
             VisualStudio.Workspace.WaitForAllAsyncOperations(Helper.HangMitigatingTimeout);
             VisualStudio.SolutionExplorer.OpenFile(project, "Class1.cs");
+            var referenceAssemblies = new ProjectUtils.PackageReference("Microsoft.NETFramework.ReferenceAssemblies.net461", "1.0.2");
+            VisualStudio.SolutionExplorer.AddPackageReference(project, referenceAssemblies);
+            VisualStudio.SolutionExplorer.RestoreNuGetPackages(project);
```

Comment:
💡 If this moves up 3 lines (above the existing call to `RestoreNuGetPackages`), the second call to `RestoreNuGetPackages` can be removed.